pkg/utils/helm/helm_helper.go

```diff
@@ -351,3 +361,44 @@ func calculateCacheTimeFromIndex(length int) time.Duration {
 	}
 	return cacheTime
 }
+
+// nolint
+func fetchChartValuesFromOciRepo(repoURL string, chartName string, version string, opts *common.HTTPOption) (string, error) {
```

Comment:
this API can be very slow:
1. any way to make it faster?
2. any cache mechanism we should do to avoid slow every time?

Reply:
1. Chart is packaged and stored in oci registry, the only way to fetch values from a chart is downloading it.
2. Outer func will cache the result.  Next time when fetch the values will use cache.